main_logic/tts_client.py

```
@@ -1098,6 +1098,292 @@ def gemini_tts_worker(request_queue, response_queue, audio_api_key, voice_id):
+    tts_config = cm.get_model_api_config('tts_custom')
+
+    ws_url = (tts_config.get('base_url') or core_config.get('GPT_SOVITS_WS_URL', '') or '').strip()
+    if not ws_url or not (ws_url.startswith('ws://') or ws_url.startswith('wss://')):
+        logger.error(f"GPT-SoVITS WebSocket URL 无效: {ws_url or '(empty)'}")
+        response_queue.put(("__ready__", True))
+        while True:
+            try:
+                sid, _ = request_queue.get()
+                if sid is None:
+                    continue
+            except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)